wallet/lib/ipc/handler/handlers/execute_recipe_handler.dart

```diff
@@ -20,8 +20,9 @@ class ExecuteRecipeHandler implements BaseHandler {
 
     final walletsStore = GetIt.I.get<WalletsStore>();
 
-    final response = await walletsStore.executeRecipe(jsonMap);
-    response.sender = sdkIpcMessage.sender;
+    final responseEither = await walletsStore.executeRecipe(jsonMap);
```

Comment:
What if any error happens and response is failure then?

Reply:
Yes handled the failure scenarios. But I want to have a short discussion with you regarding this part of code.